pipelines/batch/src/main/java/com/google/fhir/analytics/FhirEtl.java

```diff
@@ -137,6 +172,45 @@ static void fetchPatientHistory(
     }
   }
 
+  /**
+   * Initializes the ViewManager, SchemaMap. And checks if multiple View Definitions are applied to
+   * 1 resource.
+   *
+   * @param options the pipeline options
+   * @param segmentMap A mapping of FHIR Resource Type : List<SearchSegmentDescriptor>. Every
+   *     element in the list defines a set of FHIR resources to fetch in one FHIR call.
+   */
+  private static void setupViewDefinitionToParquet(
+      FhirEtlOptions options, Map<String, List<SearchSegmentDescriptor>> segmentMap) {
+    try {
+      VIEW_MANAGER = ViewManager.createForDir(options.getViewDefinitionsDir());
```

Comment:
When we switch to using `ParquetUtil` we don't need this either.

Reply:
Done.